### PR TITLE
Updating README file's dev env setup.

### PR DESCRIPTION
**Pull request by roysagi**
Updated the README file's Dev Environment Setup section in accordance with the guide I created with @reutshal and @dantavori who tested it out and it worked for them.

It should be noted that currently the requirements.txt does not include the packages needed for lint (flake8, mypy, bandit, and such) these are included in test-requirements.txt file and I added a step to install these.

We should consider uniting the two requirements files...

@liorblob - FYI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,18 +150,47 @@ You can import the SDK core class in your python code as follows:
...
-```
-Then setup dev virtual envs for python 3 (will also install all necessary requirements):
-```
-tox --devenv venv3 --devenv py37
-```
+1) Clone the Demisto-SDK repository to your existing Demisto directory. It is recommended to use SSH when cloning.
+2) Run:
+     ```
+     pip install demisto-sdk.
+     ```
```
Comment (liorblob) [inline README.md]: This is not required for development.

**Comment by teizenman** [comment]
When I tried running `tox --devenv venv3 --devenv py37` it failed on mypy installation saying `ERROR: You need Python 3.5 or later to use mypy.`

@GalRabin fixed it for me by:

1. `pip install tox-pyenv`
2. `pyenv versions`
3. `pyenv local 3.7.*`

and then the command succeeded

File: README.md [hunk 2]
```diff
@@ -150,18 +150,47 @@ You can import the SDK core class in your python code as follows:
...
-```
-List configured environments:
-```
-tox -l
-```
-Then setup dev virtual envs for python 3 (will also install all necessary requirements):
-```
-tox --devenv venv3 --devenv py37
-```
+1) Clone the Demisto-SDK repository to your existing Demisto directory. It is recommended to use SSH when cloning.
```
[the remark this answers is not in the document]
Comment (roysagi) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (roysagi) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -150,18 +150,51 @@ You can import the SDK core class in your python code as follows:
...
+
+2) **If you are using a default python version 3.7 or 3.8 you can skip this part.**
+
+    [pyenv](https://github.com/pyenv/pyenv) is an easy tool to control the versions of python on your environment.
+[Install pyenv](https://github.com/pyenv/pyenv#installation) and then run:
+    ```
+    pyenv install 3.7.5
+    pyenv install 3.8.0
+    ```
+    After installing run in `~/dev/demisto/demisto-sdk`:
```
Comment (liorblob) [inline README.md]: Are we assuming everyone's going to have this directory?

Comment (ronykoz) [inline README.md]:
```suggestion
    After installing run in `{path_to_demisto-sdk}/demisto-sdk`:
```

File: README.md [hunk 4]
```diff
@@ -150,18 +150,51 @@ You can import the SDK core class in your python code as follows:
...
+    ```
+    cd ~/dev/demisto/demisto-sdk
```
Comment (ronykoz) [inline README.md]:
```suggestion
    cd {path_to_demisto-sdk}demisto-sdk
```

**Comment by ronykoz** [comment]
@roysagi lets finish those last few touches and merge this PR please

File: README.md (hunk 4 above)
Comment (roysagi) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (roysagi) [reply README.md]: done

**Comment by roysagi** [comment]
@bakatzir - will help test this readme and then we merge

**Comment by roysagi** [comment]
Checked with @bakatzir and updated the steps accordingly.

File: README.md [hunk 5]
```diff
@@ -150,18 +150,63 @@ You can import the SDK core class in your python code as follows:
...
+  ```
+  or this command to use python 3.8:
+   ```
+ source .tox/py38/bin/activate
+ ```
+While in the virtual environment, you can use the ```demisto-sdk``` commands with all the changes made in your local environment.
+
+In case your local changes to `demisto-sdk` are not updated, you need to update your `tox` environment
+by running this command from the Demisto-SDK repository:
+```angular2
```
Comment (liorblob) [inline README.md]: angular2?

**Pull request merged**